osvr/RenderKit/RenderManagerBase.cpp

```diff
@@ -273,12 +274,14 @@ namespace renderkit {
 
         /// Construct my logger
         m_log = osvr::util::log::make_logger("RenderManager");
```

Comment:
If this makes a new log file for each RenderManager, then I think we want to go back to having one shared by all of them.  We'll want the debug messages to be interleaved in the same file.  Better to have to prefix each error with the particular RenderManager.  If they are all going to the same log file, this should be fine.